### ISSUE TITLE
Add scope attribute to tables

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#3312]
Our accessibility audit flagged missing scope attributes for tables.

## `revenue-types`
```
 • Error: Not all th elements in this table have a scope attribute. These cells should contain a scope attribute to iden
tify their association with td elements.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.H63.1
   ├── #revenue-types
   └── <table is="bar-chart-table" class="revenue table-arrow_box" id="revenue-types"> <thead> <tr> <th c...</table>
```

**Comment by mcharg** [comment]
Closing in favor of new accessibility audit.
